dev/test-zeekoe.py

```diff
@@ -40,6 +41,17 @@
 MERCH_SETUP = "merch-setup"
 CUST_SETUP = "cust-setup"
 SCENARIO = "scenario"
+TEST_ALL = "test-all"
```

Comment:
Could you add the test-all option to the examples in the comments on top?

Reply:
added!